### ISSUE TITLE
It will only be read once in the loop

### ISSUE THREAD

**Issue by sdwufei** [chrmlinux/rssRead#3]
When I put the reads RSS program into the loop
I found that only the first time it can be displayed normally
Is there any index that needs to be cleared?

[pic1](https://upload.cc/i1/2023/03/03/hU5yVz.jpg)
[pic2](https://upload.cc/i1/2023/03/03/5yebai.jpg)

**Comment by zcrowman** [comment]
Yes. I ran into this issue as well. Have been wasting a lot time trying to get it to loop properly and after seeing I'm not alone it seems that this is a limitation of this library. Time to use another one.

**Comment by Gianfilippo980** [comment]
This is the same issue that I wanted to address in my fork, if I understand it correctly.
